src/contractTest/java/uk/gov/hmcts/reform/sscs/consumer/PartiesNotifiedPutPactConsumerTest.java

```diff
@@ -0,0 +1,204 @@
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.FIELD_ID;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.PROVIDER_NAME;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.VALID_CASE_ID;
+
+@ExtendWith(PactConsumerTestExt.class)
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.reform.sscs.service"})
+@ActiveProfiles("contract")
+@SpringBootTest
+@PactTestFor(port = "10000")
+@PactFolder("pacts")
+class PartiesNotifiedPutPactConsumerTest extends BasePactTest {
+    private static final String VERSION = "version";
+    private static final String VERSION_NUMBER = "123";
+    private static final String FORBIDDEN_CASE_ID = "456";
+    private static final String NOT_FOUND_CASE_ID = "789";
+
+    @Autowired
+    private HmcHearingPartiesNotifiedApi hmcHearingPartiesNotifiedApi;
+
+    @BeforeEach
+    public void timeGapBetweenEachTest() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(2);
+    }
+
+    @Pact(provider = PROVIDER_NAME, consumer = CONSUMER_NAME)
+    public RequestResponsePact putPartiesNotifiedRequestForValidRequest(PactDslWithProvider builder) throws JsonProcessingException {
+        return builder.given(CONSUMER_NAME + " successfully putting a PartiesNotified request ")
+                .uponReceiving("Request to put PartiesNotified request")
+                .path(ContractTestDataProvider.PARTIES_NOTIFIED_PATH)
+                .method(HttpMethod.PUT.toString())
+                .query(FIELD_ID + "=" + VALID_CASE_ID + "&" + VERSION + "=" + VERSION_NUMBER)
+                .body(ContractTestDataProvider.toJsonString(ContractTestDataProvider.generatePartiesPutRequest()))
+                .headers(ContractTestDataProvider.authorisedHeaders)
+                .willRespondWith()
+                .status(HttpStatus.OK.value())
+                .toPact();
+    }
+
+    @Pact(provider = PROVIDER_NAME, consumer = CONSUMER_NAME)
+    public RequestResponsePact badRequestErrorFromPutPartiesNotified(PactDslWithProvider builder) {
+        return builder.given(CONSUMER_NAME
+                        + " throws bad request error while trying to put PartiesNotified")
+                .uponReceiving("Request to PUT PartiesNotified for bad PartiesNotified request")
+                .path(ContractTestDataProvider.PARTIES_NOTIFIED_PATH)
+                .method(HttpMethod.PUT.toString())
+                .query(FIELD_ID + "=" + VALID_CASE_ID + "&" + VERSION + "=" + VERSION_NUMBER)
+                .body(ContractTestDataProvider.toJsonString(ContractTestDataProvider.generateInvalidPartiesPutRequest()))
+                .headers(ContractTestDataProvider.authorisedHeaders)
+                .willRespondWith()
+                .status(HttpStatus.BAD_REQUEST.value())
+                .body(new PactDslJsonBody()
+                        .stringType(ContractTestDataProvider.FIELD_MESSAGE, ContractTestDataProvider.MSG_400_HEARING)
```

Comment:
can you import the static instead of using the ContractTestDataProvider qualifier